src/zap/zapinfo.cpp

```diff
@@ -1715,6 +1715,9 @@ ReadyToRunHelper MapReadyToRunHelper(CorInfoHelpFunc func, bool * pfOptimizeForS
     case corInfoHelpFunc: flags return readyToRunHelper;
 #include "readytorunhelpers.h"
 
+    case CORINFO_HELP_TYPEHANDLE_TO_RUNTIMETYPEHANDLE:
```

Comment:
This change is not related to the RVA field fix. This is a new JIT helper for CoreRT (it is redundant for CoreCLR) that was not getting mapped correctly for R2R.